CONTRIBUTING.md

```diff
@@ -17,10 +36,51 @@ The primary technologies used in this project are:
 
 - [Python](https://www.python.org)
 - [Flask](http://flask.pocoo.org)
-- [Postgres](https://www.postgresql.org)
-
-To work on the codebase for this project, you will need to have those dependencies installed.
-
+- [Docker](https://www.docker.com/get-started)
+- [Docker Compose](https://docs.docker.com/compose/gettingstarted/)
+
+To work on the codebase for this project, you will need to have those
+dependencies installed.
+
+#### Dependency Installation
+
+1. **Docker**
+   1. **Windows**: Docker on _Windows_ can be installed via
+      [Docker Desktop](https://www.docker.com/products/docker-desktop).
+   2. **Mac OS**: Docker on _Mac OS_ can be installed via
+      [Docker Desktop](https://www.docker.com/products/docker-desktop) or via
+      [Homebrew](https://brew.sh/) by doing (`brew install docker`).
+   3. **Linux**: Depending on your Linux distribution (`cat /etc/os-release`),
+      you can use your package manager on board typically to install Docker. Ex.
+      `apt-get install docker` (if Ubuntu/Debian).
+2. **Docker Compose**
+   1. **Windows**: If installed via _Docker Desktop_, it will include
+      _docker-compose_. Try to run _docker-compose version_ on commandline.
+   2. **Mac OS**: Similar to above, if installed via _Docker Desktop_, it will
+      include _docker-compose_. If using Homebrew please do
+      `brew install docker-compose`.
+   3. **Linux**: Please follow instructions at the _Linux_ tab
+      [here](https://docs.docker.com/compose/install/). Linux has shell
+      autocompletions available which are very helpful.
+
+#### Dependency Resources
+
+##### Cheatsheets
+
+Cheatsheets are a nice way to get an high at-a-glance view of the
+technology/tool. It's a very nice way to ramp up quickly with specific tooling.
+
+- Python
+  - [here](https://learnxinyminutes.com/docs/python3/)
+  - [here](https://devhints.io/python)
+- Flask
+  - [here](https://s3.us-east-2.amazonaws.com/prettyprinted/flask_cheatsheet.pdf)
+  - [here](http://flask-cheat-sheet.herokuapp.com/)
+- Docker
+  - [here](https://devhints.io/docker)
```

Comment:
I like the cheatsheet links, but I'm not a fan of calling all of them `here`. Can we replace that with the name of each cheatsheet? Example 
```
[Learn X in Y minutes](https://learnxinyminutes.com/docs/python3/)
```

Reply:
Yes, I like this idea. I have incorporated it.